src/main/java/com/amazon/opendistroforelasticsearch/security/support/Base64Helper.java

```diff
@@ -76,7 +77,7 @@
         SocketAddress.class,
         InetSocketAddress.class,
         Pattern.class,
-        User.class,
+        UserInjector.InjectedUser.class,
```

Comment:
`User` is assignable from `UserInjector.InjectedUser`, so both `User` and `UserInjector.InjectedUser` should be serializable.

Reply:
User is moved to SAFE_ASSIGNABLE_FROM_CLASSES